packages/all/package.json

```
@@ -24,15 +24,15 @@
     "clean": "rm -Rf ./node_modules ./lib/* ./meta/bundlesize/* ./meta/coverage/*",
     "prepare": "npm run build",
     "prebuild": "npm run clean",
-    "build": "../../node_modules/.bin/tsc -p ./config",
-    "postbuild": "../../node_modules/.bin/rollup -c ./config/rollup.config.js",
+    "build": "npx tsc -p ./config",
+    "postbuild": "npx rollup -c ./config/rollup.config.js",
     "predeploy": "npm run build",
     "deploy": "npm publish --tag beta"
   },
   "peerDependencies": {
     "apollo-client": "^2.6.0",
     "graphql": "^14.2.1",
-    "react": "^16.8.6"
+    "react": "^16.8.3"
```

I'm not a huge fan of basing our peerDeps on the fact that another library (React Native in this case) hasn't been updated to use the latest version. I guess in this case though, the impact is fairly harmless. If we're going to open up the range a bit though, maybe we should backup all the way to `^16.8.0`. We aren't dependent on anything in the 16.8.x patch bumps, but are dependent on 16.8.0. This way we're going back about 5 months, and would likely cover more cases than just React Native. 

```suggestion
    "react": "^16.8.0"
```

---

SGTM! should I do it in the other packages as well?